public/services/SnapshotManagementService.ts

```diff
@@ -123,6 +124,12 @@ export default class SnapshotManagementService {
     return response;
   };
 
+  catSnapshotIndices = async (indices: string): Promise<ServerResponse<CatIndex[]>> => {
```

Comment:
Just to clarify, does the response object contain an object with an array of `CatIndex` as an value, or does it just return the array of `CatIndex`? I'm trying to get an understanding of whether you'd instead want to replace the `CatIndex[]` in `ServerResponse<CatIndex[]>` with an interface similar to the [GetIndicesResponse](https://github.com/opensearch-project/index-management-dashboards-plugin/blob/main/server/models/interfaces.ts#L80).

Reply:
It contains an array of CatIndex.  I could wrap it in a new interfaces and just call it CatIndexResponse.  Although this seems more descriptive.

Reply:
Ah, I don't think that's necessary. I just wanted to clarify after seeing how `GetIndicesResponse` was used.